.github/workflows/full-build.yml

```diff
@@ -39,10 +41,14 @@ jobs:
         run: pip install babel && python po/check-babel.py
 
   linux-gtk4:
+    name: Linux (GTK4)
     needs: lint
     runs-on: ubuntu-22.04
     timeout-minutes: 30
     if: "!contains(github.event.head_commit.message, 'skip ci')"
+    outputs:
+      targz: gaphor-${{ steps.setup_and_test.outputs.version }}.tar.gz
+      wheel: gaphor-${{ steps.setup_and_test.outputs.version }}-py3-none-any.whl
```

Comment:
Do these need to have `dist/` prefixed to the name?

Reply:
No, since they're uploaded as artifacts without `dist/`. In the Publish to PyPI job it's downloading them in the `dist/` folder.